### PR TITLE
Microverse Fisrt Project

### PR DESCRIPTION
**Pull request by ifeanyi305**
My first pull request for my first repository in the microverse program. This is my Hello Microverse project and I have done the following things in it.

- Added a markup file (HTML)
- Added CSS
- GitHub flow added
- Readme file modified
- Linters installed for HTML and CSS

### CODE REVIEW

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @ifeanyi305, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Followed GitHub flow✔️

## Required Changes ♻️
- [ ] Missing linters: Please add the linter files to this repository. 
image

- [ ] Add `test.md` inside the `.gitignore` and make sure it is ignored by git and does not commit to this repository. 😄 
image


## Optional suggestions

- [OPTIONAL] Remember to always capitalize the first word of your commit messages 😄

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @ifeanyi305  

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the required changes, but you still have some issues: 

[ ] kindly add a summary to your PR (example: In This PR i ...) 🤓.

[ ] kindly Set up the linters for your repository (add the stylelint, website, and Lighthouse configuration. you can find the steps[ here](https://github.com/microverseinc/linters-config/tree/master/html-css). ( The` linter.yml` file should be inside the `.githu`b folder, and the  `hintrc` and `stylelintrc` should be in the root directory ).🤓

[ ] kindly add the test.md file inside the .gitignore file. it should be ignored by git.

[ ] kindly add an `h1` tag inside your index.html file, it should contain a "hello microverse" text.🤓
[ ] kindly add one class with styles for your header in the `style.css` file. 🤓

As they are parts of the project requirements :  👀
req

 
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (bobb-Rob) [inline index.html:1]: - [ ] [REQUIRED] This pull request(PR) can be improved by updating the title and description.
Pull Request should have a descriptive title telling what changes are inside.
Pull Request should have a summary describing introduced changes in more detail.
e.g Title -> Hello Microverse project and the summary -> Mention the things you did in brief detail.

See the image below for an example of a descriptive PR summary 👇

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @ifeanyi305 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @sameh080081, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**